src/components/Footer/styledComponents.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+
+import P from "../P";
+import { minDevice } from "../../assets/styles/breakpoints";
+import { darkGradientBackground, white } from "../../assets/styles/colors";
+
+const FooterWrapper = styled.div`
+  width: 100%;
+  height: auto;
```

Comment:
I am wondering whether this is actually needed, `auto` is the default anyway. 🤔  removing it, does not affect the UI.